ott/tools/transport.py

```diff
@@ -121,9 +121,14 @@ def solve(
   linear = isinstance(pb, linear_problems.LinearProblem)
   solver_fn = sinkhorn.make if linear else gromov_wasserstein.make
   geom_keys = ['cost_fn', 'power', 'online']
-  remove_keys = geom_keys + eps_keys if linear else geom_keys
+
+  init_dual_a = kwargs.get('init_dual_a', None)
```

Comment:
This is a good idea, I have moved them to function signature. 

It may be a good idea in future to change this solve function to remove other .get / .pop logic. 

I quite like the inspect functionality of signatures to handle multiple kwargs but I do not know what is best practice here